CPT4/vocab_specific_QA.sql

```diff
@@ -0,0 +1,65 @@
+-- 1. Our aim is to build a unique hierarchy of Procedures with CPT4 embedded in SNOMED/OMOP Ext hierarchy.
+--- The script below retrieves following data:
+---- 'concepts_in_hierarchy' - the number of concepts in hierarchy through 'Is a' relationships;
+---- 'mapped_concepts' - the number of mapped to SNOMED/OMOP Ext. concepts;
+---- 'concepts_not_in_hierarchy' - the number of concepts that are not yet in hierarchy.
+--- These counts don't include classification concepts and modifiers, and also concepts mapped to other vocabularies.
+--- Use this counts for analysis and renew respective numbers in https://github.com/OHDSI/Vocabulary-v5.0/wiki/Known-Issues-in-Vocabularies
```

Comment:
I understand, that this is postgeneric check. But can we somehow include previous numbers here to see the dynamics? They are posted on github, should we hardcode them here? Change it to pregeneric checks?
@Alexdavv @m-khitrun thoughts?

Reply:
We can add the comparison with devv5 as an additional column, like we do in manual checks after generic.